interfaces/portal/src/app/services/linguonyms/spanish.ts

```diff
@@ -0,0 +1,187 @@
+import { languageCodeToLinguonym } from '~/services/get-registration-preferrred-language-name.service';
+
+export const languageCodesToLinguonymsForSpanish: languageCodeToLinguonym = {
+  aa: 'afar',
+  ab: 'abjasio (o abjasiano)',
+  ae: 'avéstico',
+  af: 'afrikáans',
+  ak: 'akano',
+  am: 'amhárico',
+  an: 'aragonés',
+  ar: 'árabe',
+  as: 'asamés',
+  av: 'avar (o ávaro)',
+  ay: 'aimara',
+  az: 'azerí',
+  ba: 'baskir',
+  be: 'bielorruso',
+  bg: 'búlgaro',
+  bi: 'bislama',
+  bm: 'bambara',
+  bn: 'bengalí',
+  bo: 'tibetano',
+  br: 'bretón',
+  bs: 'bosnio',
+  ca: 'catalán',
+  ce: 'checheno',
+  ch: 'chamorro',
+  co: 'corso',
+  cr: 'cree',
+  cs: 'checo',
+  cu: 'eslavo eclesiástico antiguo',
+  cv: 'chuvasio',
+  cy: 'galés',
+  da: 'danés',
+  de: 'alemán',
+  dv: 'maldivo (o dhivehi)',
+  dz: 'dzongkha',
+  ee: 'ewé',
+  el: 'griego (moderno)',
+  en: 'inglés',
+  eo: 'esperanto',
+  es: 'español (o castellano)',
```

Comment:
is this intentional with lowercase letter and including the (o castellano)? This is how this shows in the language switcher now.

<img width="264" height="74" alt="Image" src="https://github.com/user-attachments/assets/5a501932-6ba5-4cdc-a160-ff40be739d48" />